middlewarex/oidc_test.go

```diff
@@ -0,0 +1,81 @@
+package middlewarex
+
+import (
+	"context"
+	"net/http"
+	"net/http/httptest"
+	"testing"
+
+	"github.com/coreos/go-oidc/v3/oidc"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestValidateAuthHeader(t *testing.T) {
+	tests := []struct {
+		authHeader    string
+		scheme        string
+		expectedToken string
+		expectedOk    bool
+	}{
+		{
+			authHeader: "", scheme: "bearer",
+			expectedToken: "", expectedOk: false,
```

Comment:
Can you change this to `expectedOK`? Example from Go’s standard library: `http.StatusOK`.